src/main/java/org/betonquest/betonquest/compatibility/holograms/HologramLoop.java

```diff
@@ -90,6 +90,8 @@ private HologramWrapper initializeHolograms(final int defaultInterval, final Que
         } catch (final NumberFormatException e) {
             throw new InstructionParseException("Could not parse check interval", e);
         }
+        final int maxRange = parseMaxRange(section.getString("max_range"));
```

Comment:
```suggestion
        final VariableNumber maxRange = new VariableNumber(pack, section.getString("max_range", "0"));
```
I would suggest to use variableNumber, but you need to fix some ther parts then

If you dont like to do this, you should at least use `getInt()` instead of `getString()`